floodviz/static/js/hydrograph.js

```diff
@@ -1,17 +1,19 @@
-document.addEventListener("DOMContentLoaded", function (event) {
-	"use strict";
+"use strict";
 
-	var data_path = FV.hydrograph_data_path;
+FV.hydromodule = function (options) {
```

Comment:
Need to add comments describing what this does and what properties options expects and their types as well as whether any are optional.